syntax = "proto2";

package rv.protos;

import "rastervision/protos/raster_source.proto";
import "rastervision/protos/label_store.proto";

message Scene {
    optional RasterSource raster_source = 1;

    // In order to generate training data from a scene, or to evaluate a
    // model on a scene, the ground_truth_label_store needs to be
    // specified.
    optional LabelStore ground_truth_label_store = 2;

    // In order to make a prediction on a scene, the
    // prediction_label_store needs to be specified.
    optional LabelStore prediction_label_store = 3;

    // The id is used by the workflow runner to generate some output URIs.
    optional string id = 4;
}
